File: src/main/Python/boj_step10/boj9063.py
import sys
cin = sys.stdin.readline

N = int(cin())
xList = []
yList = []
for _ in range(N):
    x, y = map(int, cin().split())
    xList.append(x)
    yList.append(y)
print((max(xList) - min(xList)) * (max(yList) - min(yList)))

File: src/main/Cpp/boj_step10/boj9063.cpp
#include <iostream>
#include <list>
#include <algorithm>
using namespace std;

int main() {
    int N;
    list<int> xList;
    list<int> yList;

    cin >> N;

    for (int i = 0; i < N; i++) {
        int x, y;
        cin >> x >> y;
        xList.push_back(x);
        yList.push_back(y);
    }

    int width = *max_element(xList.begin(), xList.end()) - *min_element(xList.begin(), xList.end()),
    height = *max_element(yList.begin(), yList.end()) - *min_element(yList.begin(), yList.end());

    cout << width * height;
}